release-notes/7.0/releases.json

```diff
@@ -1,14 +1,709 @@
 {
   "channel-version": "7.0",
-  "latest-release": "7.0.19",
-  "latest-release-date": "2024-05-14",
-  "latest-runtime": "7.0.19",
-  "latest-sdk": "7.0.409",
-  "release-type" : "sts",
-  "support-phase": "eol",
+  "latest-release": "7.0.20",
+  "latest-release-date": "2024-05-28",
+  "latest-runtime": "7.0.20",
+  "latest-sdk": "7.0.410",
+  "support-phase": "maintenance",
```

Comment:
```suggestion
  "support-phase": "eol",
```

Reply:
Wandering through end-of-life experiences, as grateful as I am confused, with heavy bags and no exact destination is the true background noise behind this comment. My goal moving forward is to simplify, restore, and execute tasks more efficiently on the internet. Focusing time to specific objectives rather than aimlessly pushing buttons on unfamiliar applications, seems like a logical route to more production, and better usage of time. Thank you for listening, but it just dawned on me how there is always another puzzle. Like how to add hash correctly, which was my question, versus hitting pause on coding to focus on documentation. Appreciate you being a sounding board to clear my head but perhaps I am off topic now... have a great evening.